### ISSUE TITLE
Shouldn't L197 be changed?

### ISSUE THREAD

**Issue by lhorace** [jprjr/multistreamer-installer#9]
https://github.com/jprjr/multistreamer-installer/blob/37fd141a0c8d6b3fa12e6b3a33f2e1a13e33d7dc/install#L197

Shouldn't this line be change, there seems to be differences in rockspecs?

```
 diff -u multistreamer-dev-1.rockspec multistreamer-12.2.2-0.rockspec
--- multistreamer-dev-1.rockspec	2019-01-06 08:35:44.941432367 -0500
+++ multistreamer-12.2.2-0.rockspec	2019-02-25 07:47:22.411487355 -0500
@@ -1,9 +1,9 @@
 package = "multistreamer"
-version = "dev-1"
+version = "12.2.2-0"
 
 source = {
-  url = "https://github.com/jprjr/multistreamer/archive/luarocks.tar.gz",
-  file = "multistreamer-luarocks.tar.gz",
+  url = "https://github.com/jprjr/multistreamer/archive/12.2.2.tar.gz",
+  file = "multistreamer-12.2.2.tar.gz",
 }
 
 dependencies = {
@@ -37,6 +37,7 @@
       ["multistreamer.config"] = "lib/multistreamer/config.lua",
       ["multistreamer.getopt"] = "lib/multistreamer/getopt.lua",
       ["multistreamer.http"] = "lib/multistreamer/http.lua",
+      ["multistreamer.importexport"] = "lib/multistreamer/importexport.lua",
       ["multistreamer.migrations"] = "lib/multistreamer/migrations.lua",
       ["multistreamer.nginx-conf"] = "lib/multistreamer/nginx-conf.lua",
       ["multistreamer.process_manager"] = "lib/multistreamer/process_manager.lua",
```

**Comment by jprjr** [comment]
Oh good catch, I forgot to add that 'importexport' module to the "dev" rockspec.

But yeah, since the script is installing a specific version of multistreamer anyway, I should be able to point to a specific rockspec. I just need to make sure I include the rockspec in the tagged release

**Comment by jprjr** [comment]
Fixed in 7def28221cca500db673a2c3e8b668f846bb826c
